src/angular/projects/spark-extras-angular-card/package.json

```diff
@@ -19,7 +19,7 @@
     "@angular/platform-browser": "^6.1.4",
     "@angular/platform-browser-dynamic": "^6.1.4",
     "@angular/router": "^6.1.4",
-    "@sparkdesignsystem/spark-card": "^4.0.1",
-    "@sparkdesignsystem/spark-core-angular": "^7.0.0"
+    "@sparkdesignsystem/spark-card": "^4.0.2",
```

Comment:
Card is at 4.0.2 and this was outdated. https://www.npmjs.com/package/@sparkdesignsystem/spark-card